website_sale_stock_available/i18n/nl.po

```diff
@@ -0,0 +1,38 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* website_sale_stock_available
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2020-12-15 13:19+0000\n"
+"Last-Translator: Bosd <c5e2fd43-d292-4c90-9d1f-74ff3436329a@anonaddy.me>\n"
+"Language-Team: none\n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.3.2\n"
+
+#. module: website_sale_stock_available
+#: model:ir.model,name:website_sale_stock_available.model_product_template
+msgid "Product"
+msgstr "Product"
+
+#. module: website_sale_stock_available
+#: model:ir.model,name:website_sale_stock_available.model_product_product
+msgid "Product Variant"
+msgstr ""
+
+#. module: website_sale_stock_available
+#: model:ir.model,name:website_sale_stock_available.model_sale_order
+msgid "Sales Order"
+msgstr ""
+
+#~ msgid "Product Template"
+#~ msgstr "Product Sjabloon"
+
+#~ msgid "Sale Order"
+#~ msgstr "Verkoop order"
```

Comment:
Delete old translations

Reply:
It's not done by the bot. They are useless and it's only a cleanup, but a welcome one if you want to do it.